cpp/src/utilities/spmv_1D.cu

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "spmv_1D.cuh"
+
+namespace cugraph {
+namespace opg {
+template <typename VT, typename ET, typename WT>
+OPGcsrmv<VT, ET, WT>::OPGcsrmv(const raft::comms::comms_t &comm_,
+                               VT *local_vertices_,
+                               VT *part_off_,
+                               ET *off_,
+                               VT *ind_,
+                               WT *val_,
+                               WT *x)
+  : comm(comm_),
+    local_vertices(local_vertices_),
+    part_off(part_off_),
+    off(off_),
+    ind(ind_),
+    val(val_)
+{
+  stream = nullptr;
+  i      = comm.get_rank();
+  p      = comm.get_size();
+  v_glob = part_off[p - 1] + local_vertices[p - 1];
+  v_loc  = local_vertices[i];
+  VT tmp;
+  cudaMemcpy(&tmp, &off[v_loc], sizeof(VT), cudaMemcpyDeviceToHost);
+  CUDA_CHECK_LAST();

```

Comment:
No need for this if we guard the above with CUDA_TRY

Reply:
2d430fc